cli/script/command-parser.ts

```diff
@@ -29,6 +29,16 @@ function showHelp(showRootDescription?: boolean): void {
     }
 }
 
+function accessKeyAdd(commandName: string, yargs: yargs.Argv): void {
+    isValidCommand = true;
+    yargs.usage(USAGE_PREFIX + " access-key " + commandName + " --description <description>")
+        .demand(/*count*/ 2, /*max*/ 2)  // Require exactly two non-option arguments.
+        .example("access-key " + commandName + " --des \"VSO Integration\"", "Generates a new access key with the description \"VSO Integration\"")
+        .option("description", { alias: "des", demand: true, description: "A short description to be tagged to this access key", type: "string" });
```

Comment:
Nowhere else do we use required options.  To date required options have been positional arguments.